docs/timeseries/index.rst

```diff
@@ -139,8 +148,11 @@ sliced using index notation::
     2009-05-02T00:43:38.045  6.631103e-04 ...  1.5665225e-03 -1.4616371e-03
     2009-05-02T00:44:36.894  6.631350e-04 ...  1.5586632e-03 -1.4692718e-03
 
+.. EXAMPLE END
+
+.. EXAMPLE START: Accessing the Time Column in TimeSeries Objects
```

Comment:
this example seems to be a duplicate (at least based on the title, I didn't check the content):
https://github.com/astropy/astropy/pull/10147/files#diff-cc461b39bc2454172d9f5019a1554cfcR68

I suggest to only keep one of them for the example gallery and add a comment into the rst file that the other is not examplified as it was a duplicate.

Reply:
Will do

Reply:
It looks like the one in data_access is just the first snippet copied from this one, which is more extensive, so I'll leave this one and mark the other as a duplicate.